api/src/main/resources/underlays/aou_synthetic.yaml

```diff
@@ -1072,39 +1078,142 @@ hierarchies:
     descendantsTable:
       ancestor:
         dataset: omop_indexes
-        table: flatten_snomed_conditions_0
+        table: concept_ancestor_descendant_1
         column: ancestor
       descendant:
         dataset: omop_indexes
-        table: flatten_snomed_conditions_0
+        table: concept_ancestor_descendant_1
         column: descendant
     childrenTable:
       parent:
+        dataset: aou_synthetic
+        table: concept_relationship
+        column: concept_id_1
+      child:
+        dataset: aou_synthetic
+        table: concept_relationship
+        column: concept_id_2
+      tableFilter:
+        binaryColumnFilter:
+          column:
+            dataset: aou_synthetic
+            table: concept_relationship
+            column: relationship_id
+          operator: EQUALS
+          stringVal: Subsumes
```

Comment:
I chose `Subsumes` because:
- The [previous SQL query](https://github.com/DataBiosphere/tanagra/blob/8bb26c00459fc3aafc75499c79859786ac918bdf/workflow/src/main/resources/queries/aou_omop_snomed_condition_relations.sql#L10) for generating the ancestor-descendant index table for conditions filtered on this relationship name.
- Comparing SQL query results with the AoU-CB behavior seemed to mostly match when I used `relationship_id='Subsumes'`.
- An example of using `Subsumes` in the [OMOP documentation](https://ohdsi.github.io/CommonDataModel/dataModelConventions.html#Concept_Relationships) seems to match what we want for the hierarchy definition.

I'll add this to the PR description also.